code/modules/hydroponics/hydro_tray.dm

```diff
@@ -619,6 +619,59 @@
 	if(..())
 		return 1
 
+	if(istype(O, /obj/item/claypot))
+		user << "<span class='warning'>You must place the pot on the ground and use a spade on \the [src] to make a transplant.</span>"
+		return
+
+	if(seed && istype(O, /obj/item/weapon/pickaxe/shovel/spade))
```

Comment:
i'm not digging up a banana tree with a spade.  we're already operating under a high level of improbability, I don't think shovels are going to shatter the emulsions as much as the lack of shovels will frustrate the users at the inconsistency.